.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+## PR Type
+What kind of change does this PR introduce?
+
+<!-- Please check the one that applies to this PR using "x". -->
+```
+[ ] Bugfix
+[ ] Feature
+[ ] Code style update (formatting)
+[ ] Refactoring (no functional changes, no api changes)
+[ ] Build or CI related changes
+[ ] Documentation content changes
+[ ] Sample app changes
+[ ] Other... Please describe:
+```
+
+
+## What is the current behavior?
+<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
+Issue Number: N/A
+
+
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tested code with SDK 14393 and SDK 15063
```

Comment:
Shouldn't we add a reference(link) to current release min and target instead of changing this values with every release ?
Do we have this reference ? 😄

Reply:
Great suggestion, moved it readme.md